packaging/systemd/microshift-containerized

```diff
@@ -2,7 +2,5 @@
 
 set -euxo pipefail
 
-[[ -d /etc/microshift-containerized ]] || mkdir -p /etc/microshift-containerized
-cat <<EOF > /etc/microshift-containerized/microshift-containerized.conf
-export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig
-EOF
+setsebool -P container_manage_cgroup true
+sudo mkdir -p /var/lib/microshift
```

Comment:
Can we fold this file completely into the `microshift-containerized.service`, such that the UX is like we document in https://microshift.io/docs/getting-started/ ?